content/source/docs/cloud/users-teams-organizations/api-tokens.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "cloud"
+page_title: "API Tokens - Terraform Cloud"
+sidebar_current: "docs-cloud-users-teams-organizations-api-tokens"
+---
+
+# API Tokens
+
+Terraform Cloud supports three distinct types of API tokens with varying levels of access: user, team, and organization. There are differences in access levels and generation workflows for each of these token types, which are outlined below.
+
+Like with [user tokens](./users.html#api-tokens), team and organization API tokens are displayed only once when they are created, and are obfuscated thereafter. If the token is lost, it must be regenerated. When a token is regenerated, the previous token immediately becomes invalid.
```

Comment:
This feels out of place here; maybe after the sections below?  Or maybe it's just the 'Like with user tokens' part, and redoing like this would work:

> API tokens are displayed only once when they are created, and are obfuscated thereafter. If the token is lost, it must be regenerated. With a team or an organization token, the previous token immediately becomes invalid.

Reply:
(I know you probably just copied this sections that already existed)

Reply:
Yep good call, these came byte-for-byte from a different file, but now is a good time to fix this up.